### PR TITLE
Camp Supply Requirements feature

### LINKED ISSUE #325: CAMP requirements
We need to add another page where we can view information about a camp.

- Need taluk and village options in camp registering. 
- Click on a camp name to view additional details
- Text field for "Requirements in Camp"
- Number of people for each gender

Edit: Taluk and village was added

### PR DESCRIPTION
**Pull request by xaneem**
### Issue Reference
This PR addresses the Issue : Fixes #325 

### Summarize
- Relief Camp Coordinator page
- Edit Requirements list
- People Registration

Testing instructions
- Run migrations
- Create super user if you haven't already
- Create relief camp from Django Admin and create a user for relief camp (data entry user)
- Log in as the created user 
- Test person registration and edit requirements

### CODE REVIEW

**Comment by riginoommen** [comment]
Code looks clean

File: mainapp/views.py [hunk 1]
```diff
@@ -223,29 +227,77 @@ class Meta:
...
+           'sanitary_req': forms.Textarea(attrs={'rows':3}),
+           'other_req': forms.Textarea(attrs={'rows':3}),
+       }
+
+class CampDetails(SuccessMessageMixin,LoginRequiredMixin,UpdateView):
+    login_url = '/login/'
+    model = RescueCamp
+    template_name='mainapp/camp_details.html'  
+    form_class = CampDetailsForm
+    success_url = '/coordinator_home/'
```
Comment (tachyons) [inline mainapp/views.py]: `/` suffix needed ?

**Comment by vrthra** [comment]
Testing on FF seems to work

File: mainapp/templates/mainapp/coordinator_home.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  <h4>
+      No relief camps configured for this user yet.
+  </h4>
+  
+</div>
+{% endif %}
+
+
+
+</script>
```
Comment (jinojossy93) [inline mainapp/templates/mainapp/coordinator_home.html]: Is this needed?

**Comment by bhumilharia** [comment]
I've tested only on Chrome (desktop and mobile). This change works as expected, except for one tiny mobile number display issue that @xaneem is working on fixing. Good to go from my end

**Comment by xaneem** [comment]
@jinojossy93 @bhumilharia Fixed both. Please confirm and approve.

**Comment by bhumilharia** [comment]
Works as expected, approved.

File: mainapp/views.py (hunk 1 above)
Comment (tachyons) [reply mainapp/views.py]: context https://github.com/IEEEKeralaSection/rescuekerala/pull/294

Comment (xaneem) [reply mainapp/views.py]: #294 says we need trailing / or use {% url %}

**Pull request merged**